src/components/NFTPortfolio/MainView.vue

```diff
@@ -44,6 +44,44 @@
           }
         ]"
       >
+        <Dropdown v-if="isPortfolioTabCollectedActive">
+          <template v-slot:trigger="{ toggle }">
+            <ButtonV2
+              :text="$t('filter_menu_creator')"
+              preset="plain"
+              @click="toggle"
+            />
+          </template>
+          <MenuList>
+            <MenuItem
+              value=""
+              :label="$t('filter_menu_reset')"
+              label-align="left"
```

Comment:
I prefer text align center in this case
```suggestion
              label-align="center"
```